### ISSUE TITLE
Fix security issues found by github

### ISSUE THREAD

**Issue by celian-garcia** [prometheus-community/monaco-promql#5]
```
remote:
remote: GitHub found 4 vulnerabilities on prometheus-community/monaco-promql's default branch (2 high, 2 moderate). To find out more, visit:
remote:      https://github.com/prometheus-community/monaco-promql/network/alerts
remote:
```
https://github.com/prometheus-community/monaco-promql/network/alerts

**Comment by celian-garcia** [comment]
Angular is also finding 4 high vulnerabilities, it has chances to be the same ones.

```
added 1446 packages from 706 contributors and audited 905287 packages in 199.74s
found 38955 vulnerabilities (38930 low, 21 moderate, 4 high)
  run `npm audit fix` to fix them, or `npm audit` for details 
```
Will ``npm audit fix`` be the magic command ?

**Comment by celian-garcia** [comment]
Fixed by #29
